test-cases/longevity/longevity-1TB-3days-authorization-and-tls-ssl.yaml

```diff
@@ -0,0 +1,45 @@
+test_duration: 5020
+prepare_write_cmd: ["cassandra-stress write cl=ALL n=275050075 -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=150 -col 'size=FIXED(200) n=FIXED(5)' -pop seq=1..275050075",
+                    "cassandra-stress write cl=ALL n=275050075 -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=150 -col 'size=FIXED(200) n=FIXED(5)' -pop seq=275050076..550100150",
+                    "cassandra-stress write cl=ALL n=275050075 -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=150 -col 'size=FIXED(200) n=FIXED(5)' -pop seq=550100151..825150225",
+                    "cassandra-stress write cl=ALL n=275050075 -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=150 -col 'size=FIXED(200) n=FIXED(5)' -pop seq=825150226..1100200300"]
```

Comment:
After the tests @fgelcer ran last week, I'm not sure if this is still valid way to go.
@Mark-Gurevich, how long does it take to populate this 1TB and how much space does it use after the prepare stage finish?
(Check in monitor)

Let's in parallel, run another test just for prepare, with prepare from 1 loader seq=1.. 1100200300.
I want to see how long it takes and what the disk impact (space used).
Otherwise, we may hit ENOSPC with some nemesis.

Reply:
I ran the job with 1 loader and only prepare_write_cmd (stress_cmd was commented out) with seq=1.. 1100200300.
It took approximately 1 hour and 30 minutes.
Sorry, I'm not sure about the disk space impact.
Argus: https://argus.scylladb.com/workspace?state=WyI3OWUyZTIzMy00YjMyLTRmOGQtYTM5NC1mNzE5MWE2YTkxOTMiXQ

Reply:
First, it takes even less than with 4 loaders which is great.
Second, regarding disk space impact, I'm restoring the monitors of this run and previous and I'll check it out and share.

Reply:
I'll summarize in comment below that won't be disappeared.